### PR TITLE
Update responsive_amp@es.md

### PR DESCRIPTION
**Pull request by SidVal**
Updated file with changes from responsive_amp.md

### CODE REVIEW

File: content/docs/guides/responsive_amp@es.md [hunk 1]
```diff
@@ -86,25 +63,44 @@ usando propiedades comunes de CSS. Por ejemplo:
 </body>
 [/sourcecode]
 
-**Importante:**
+{% call callout('Important', type='caution') %}
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: For the English word "Important", can you change it to the Spanish word, which I believe is "Importante"?

File: content/docs/guides/responsive_amp@es.md [hunk 2]
```diff
@@ -60,13 +36,14 @@ Por ejemplo:
         background-color: grey;
       }
     </style>
-    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    ...
   </head>
 [/sourcecode]
 
-**Importante:**
+{% call callout('Important', type='caution') %}
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: For the English word "Important", can you change it to the Spanish word, which I believe is "Importante"?

File: content/docs/guides/responsive_amp@es.md [hunk 3]
```diff
@@ -1,54 +1,30 @@
...
-    src="/img/narrow.jpg"
-    srcset="/img/wide.jpg 640w,
-           /img/narrow.jpg 320w"
-    width="1698"
-    height="2911"
-    layout="responsive"
-    alt="an image">
-</amp-img>
-[/sourcecode]
+{% call callout('Tip', type='success') %}
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: For the English word "Tip", can you change it to the Spanish equivalent?

File: content/docs/guides/responsive_amp@es.md [hunk 4]
```diff
@@ -86,25 +63,44 @@ usando propiedades comunes de CSS. Por ejemplo:
...
-que se puede escalar con el contenedor.
+que puede escalar con el contenedor.
 
-Establece el diseño como adaptable.
+Configure el diseño como adaptable (responsive).
 Esto ajusta el tamaño del elemento a la anchura del elemento contenedor
 y cambia la altura automáticamente a la relación de aspecto determinada por los atributos de anchura y altura.
 
-Obtén más información acerca de [los diseños compatibles con AMP](/es/docs/guides/author-develop/responsive/control_layout.html).
+{% call callout('Read on', type='read') %}
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: For the English text "Read on", can you change it to the Spanish equivalent?

File: content/docs/guides/responsive_amp@es.md [hunk 5]
```diff
@@ -86,25 +63,44 @@ usando propiedades comunes de CSS. Por ejemplo:
...
+Aprende más acerca de [los diseños compatibles con AMP](/es/docs/guides/responsive/control_layout.html)
+{% endcall %}
+
+
+## Proporcionar placeholders & fallbacks
+
+El soporte incorporado para placeholders y fallbacks significará que los usuarios nunca tendrán que mirar una pantalla en blanco de nuevo.
+
+{% call callout('Read on', type='read') %}
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: Same as above comment

File: content/docs/guides/responsive_amp@es.md [hunk 6]
```diff
@@ -86,25 +63,44 @@ usando propiedades comunes de CSS. Por ejemplo:
...
+Aprende más sobre [Placeholders y fallbacks](/es/docs/guides/responsive/placeholders.html)
+{% endcall %}
+
+
+## Dirige tus imágenes
+
+AMP admite los atributos `srcset` y `sizes` para darle un control fino sobre cuáles imágenes se cargarán en qué escenario.
+
+{% call callout('Read on', type='read') %}
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: Here too.

File: content/docs/guides/responsive_amp@es.md [hunk 7]
```diff
@@ -119,5 +115,6 @@ Ejemplo de error en la consola de una página con CSS que excede el límite de 5
 
 <amp-img src="/static/img/docs/too_much_css.png" width="1404" height="334" layout="responsive"></amp-img>
 
-Obtén más información acerca de cómo [validar las páginas de AMP](/es/docs/guides/debug/validate.html),
-incluida la forma de localizar los errores de estilo y solucionarlos.
+{% call callout('Read on', type='read') %}
```
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: And here. :)

File: content/docs/guides/responsive_amp@es.md (hunk 3 above)
Comment (SidVal) [inline content/docs/guides/responsive_amp@es.md]: "Tip" it's just fine :+1:

File: content/docs/guides/responsive_amp@es.md (hunk 2 above)
Comment (SidVal) [inline content/docs/guides/responsive_amp@es.md]: Ok. I did not know if that could be translated.

Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: Just that first text - it's the word that displays in the callout like:
selection_005

File: content/docs/guides/responsive_amp@es.md (hunk 3 above)
Comment (bpaduch) [inline content/docs/guides/responsive_amp@es.md]: Ah okay.

File: content/docs/guides/responsive_amp@es.md (hunk 1 above)
Comment (SidVal) [inline content/docs/guides/responsive_amp@es.md]: Try now ;)

**Comment by SidVal** [comment]
Let's see now :+1:

**Pull request merged**